update bug report issue template - include PL version (#8209)
PyTorchLightning/pytorch-lightning: .github/ISSUE_TEMPLATE/bug_report.md

python collect_env_details.py
```

>  - PyTorch Lightning Version (e.g., 1.3.0):
>  - PyTorch Version (e.g., 1.0)

Reviewer: nit: we should pick a PyTorch version that Lightning supports for the template default 
```suggestion
 - PyTorch Lightning Version (e.g., 1.3.0):
 - PyTorch Version (e.g., 1.8)
```

 - Python version:
 - OS (e.g., Linux):
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - How you installed PyTorch (`conda`, `pip`, source):

>  - Build command you used (if compiling from source):

Reviewer 2: ```suggestion
 - If compiling from source, the output of `torch.__config__.show()`:
```
